services/offline/src/__tests__/integration.test.js

```diff
@@ -0,0 +1,22 @@
+import { render } from '@testing-library/react'
```

Comment:
Currently just the offlineInterface initialization is tested. More to come in the future